grug/testing/tests/httpd.rs

```diff
@@ -238,3 +248,88 @@ async fn graphql_returns_messages() -> anyhow::Result<()> {
 
     Ok(())
 }
+
+#[tokio::test(flavor = "multi_thread", worker_threads = 4)]
+async fn graphql_subscribe_to_block() -> anyhow::Result<()> {
+    let (httpd_context, mut suite, mut accounts) = create_block().await?;
+
+    let graphql_query = r#"
+      subscription Block {
+        block {
+          blockHeight
+          createdAt
+          hash
+          appHash
+        }
+      }
+    "#;
+
+    let request_body = GraphQLCustomRequest {
+        name: "block",
+        query: graphql_query,
+        variables: Default::default(),
+    };
+
+    let (crate_block_tx, mut rx) = mpsc::channel::<u32>(1);
+
+    // Can't call this from LocalSet so using channels instead.
+    tokio::spawn(async move {
+        while rx.recv().await.is_some() {
+            let to = accounts["owner"].address;
+
+            suite
+                .send_message_with_gas(
+                    &mut accounts["sender"],
+                    2000,
+                    grug_types::Message::transfer(
+                        to,
+                        Coins::one(Denom::from_str("ugrug")?, 2_000).unwrap(),
```

Comment:
We're mixing up error handling in tests. In some places we use `unwrap` while elsewhere we use `?`.